build/wd_test.bzl

```diff
@@ -47,14 +47,15 @@ def _wd_test_impl(ctx):
         ctx.actions.write(
             output = executable,
             # PowerShell correctly handles forward slashes in executable paths generated by Bazel (e.g. "bazel-bin/src/workerd/server/workerd.exe")
+            # TODO(now): How do we pass TEST_TMPDIR? I don't know powershell.
             content = "powershell -Command \"%*\"\r\n",
```

Comment:
PowerShell has a global `$ENV` variable where all of the environment variables live - like `$ENV:PATH`.

In this instance, I suspect it's something like this (minus escaping the quotes for Bazel):

```
powershell -Command "%* `-dTEST_TMPDIR=$ENV:TEST_TMPDIR"
```

Notably, the `-` for `-d` needs to be escaped by a backtick because that's a special character in PowerShell otherwise the variable won't expand.

```cmd
PS C:\Users\K> $ENV:TEST_TMPDIR = "foo"
PS C:\Users\K> .\test.bat echo

C:\Users\K>powershell -Command "echo `-dTEST_TMPDIR=$ENV:TEST_TMPDIR"
-dTEST_TMPDIR=foo
```

Reply:
With a Python script to print `sys.argv[1:]`

```
PS C:\Users\K> .\test.bat python3 .\args.py -testing=123

C:\Users\K>powershell -Command "python3 .\args.py -testing=123 `-dTEST_TMPDIR=$ENV:TEST_TMPDIR"
['-testing=123', '-dTEST_TMPDIR=foo']
```

This is all assuming that Windows is also okay with the `-dFOO=BAR` syntax.

Reply:
Thanks for the tip! Wow the use of backtick as an escape character really weirds me out.

Reply:
It's the escape character, used for escape sequences and also the equivalent of `\` in Shell where you can use it to signal that the command continues on the next-line.

PowerShell is a fun one 😅